From 678678fe0a552e565ff9e224875ecb11df34ca69 Mon Sep 17 00:00:00 2001
From: Marian-Cristian Rotariu <marian.rotariu@freescale.com>
Date: Thu, 22 Oct 2015 18:20:43 +0300
Subject: [PATCH 106/146] dpaa_eth: remove jumbo frame optimization

This optimization cannot be used on ARM platforms with the 4K hardware
limitation since it determines the creation of more that 4K sized buffers.

Signed-off-by: Marian-Cristian Rotariu <marian.rotariu@freescale.com>
Signed-off-by: Madalin Bucur <madalin.bucur@freescale.com>
---
 drivers/net/ethernet/freescale/sdk_dpaa/Kconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig b/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig
index 9af50cc..8857662 100644
--- a/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig
+++ b/drivers/net/ethernet/freescale/sdk_dpaa/Kconfig
@@ -56,6 +56,7 @@ config FSL_DPAA_GENERIC_DRIVER
 
 config FSL_DPAA_ETH_JUMBO_FRAME
 	bool "Optimize for jumbo frames"
+	depends on !ARM64
 	default n
 	---help---
 	  Optimize the DPAA Ethernet driver throughput for large frames
@@ -65,6 +66,7 @@ config FSL_DPAA_ETH_JUMBO_FRAME
 	  Using this option in combination with small frames increases
 	  significantly the driver's memory footprint and may even deplete
 	  the system memory.
+	  This option is not available on LS1043.
 
 config FSL_DPAA_TS
 	bool "Linux compliant timestamping"
-- 
2.1.0.27.g96db324

